libraries/node-core-library/src/FileSystem.ts

```diff
@@ -521,4 +812,42 @@ export class FileSystem {
   public static getRealPath(linkPath: string): string {
     return fsx.realpathSync(linkPath);
   }
+
+  /**
+   * An async version of {@link FileSystem.getRealPath}.
+   */
+  public static getRealPathAsync(linkPath: string): Promise<string> {
+    return fsx.realpath(linkPath);
+  }
+
+  // ===============
+  // UTILITY FUNCTIONS
+  // ===============
+
+  /**
+   * Returns true if the error provided indicates the file or folder
+   * does not exist.
+   */
+  public static isNotExistError(error: NodeJS.ErrnoException): boolean {
```

Comment:
Maybe this should be `@internal`.  (?) It seems not super useful to external consumers.

Reply:
I think this actually is pretty useful. There are places I want to use it in other tools. I'd like to include it.